source/guide/3.2/webui-blueprints-page.md

```diff
@@ -13,15 +13,31 @@ terminology_link: reference-terminology.html
 When clicking on the `Blueprints` tab and choosing a blueprint you will be able to choose one of the following:
 
 # Topology
-See the definition [here]({{page.terminology_link}}#topology).<br/>
```

Comment:
I see that this description is pretty much cloned in the blueprints page and deployments page. Maybe we can deduplicate somehow?